libs/utils/android/workloads/uibench.py

```diff
@@ -84,6 +84,11 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         # Keep track of mandatory parameters
         self.out_dir = out_dir
         self.collect = collect
+        # For systrace collection, the duration is expected to be part of `collect` parameter
+        # This is because systrace capturing currently doesnt support CTRL+C stopping
```

Comment:
Actually it seems that systrace can be "stopper on demand"... by just running it on background and than sending an ENTER when we are done:
https://github.com/catapult-project/catapult/blob/master/systrace/systrace/run_systrace.py#L169

I've just added this as an issue: #331